src/Internal/EventStoreNodeConnection.php

```diff
@@ -581,6 +586,7 @@ public function createPersistentSubscriptionAsync(
         return $deferred->promise();
     }
 
+    /** {@inheritdoc} */
```

Comment:
inheritdoc or copy paste the phpdoc block?

Reply:
I think the tools are using inheritdoc implicitly now so nothing should be necessary... but I might be wrong.